python/src/cloudstorage/cloudstorage_api.py

```diff
@@ -273,6 +275,185 @@ def listbucket(path_prefix, marker=None, prefix=None, max_keys=None,
   return _Bucket(api, bucket, options)
 
 
+# pylint: disable=too-many-locals, too-many-branches, too-many-statements
+def compose(list_of_files, destination_file,
+            content_type=None, retry_params=None, _account_id=None):
+  '''
+    Runs the GCS Compose on the inputed files.
+    Merges between 2 and 32 files into one file. If any of the previous files had the merge
+    run on them it counts towards the 32 limit.
+  Args:
+    list_of_files: list of file name strings or dictionaries with the following format:
```

Comment:
Response to second comment:
So 2 variables list_of_files (list of strings) and files_metadata (list of dictionaries). Then merge them in code?